File: Java Practice/Ex 2/fizzbuzz.java
public class fizzbuzz {
    public static void main(String[] args) {
        int n = 50;

        if( n % 3 == 0 && n % 5 != 0){
            System.out.println("Fizz");
        }
        else if(n % 5 == 0 && n % 3 != 0){
            System.out.println("Buzz"); 
        }
        else if(n % 3 == 0 && n % 5 == 0){
            System.out.println("FizzBuzz");
        }
        else {System.out.println(n);
        }

    }
}

File: Computational Linguistics/Sem 1/Computational Processing of Speech Rhythm/R Basic Practice/Basic Exercises/fizzbuzz.r
fizzbuzz <- function(upper){
    results <- character(upper)

    if(upper < 1){
        stop("Upper bound must be greater than 0")
    }

    for(each in 0:upper){
        if(each %% 3 == 0 && each %% 5 == 0){
            results[each] <- "fizzbuzz"
        }
        else if(each %% 3 == 0){
            results[each] <- "fizz"
        }
        else if(each %% 5 == 0){
            results[each] <- "buzz"
        }
        else{
            results[each] <- as.character(each)
        }
    }

    return(results)
}

result <- fizzbuzz(10)
cat(result, sep = "\n")